ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml

```diff
@@ -28,6 +28,11 @@ ref:
     default: "m5.xlarge"
     documentation: |-
       The instance type to use for compute nodes (e.g. AWS https://aws.amazon.com/ec2/instance-types/). We use a 4 core worker to match the median configuration of the fleet.
+  - name: MASTER_ARCH
```

Comment:
Maybe "ARCH_TYPE" sounds more reasonable?

Reply:
This variable only handles the control plane architecture. Hence, I used MASTER_ARCH. 

I could use something like the "COMPUTE_NODE_TYPE" variable for the workers. However, the node size of the control plane is managed through the SIZE_VARIANT variable. 

See row 31 (hidden) of https://github.com/openshift/release/pull/28147/files#diff-e12f1cf79ea6ed6c844819d0d23955fdaa6267109829daeeaa38237dce596532

Reply:
Per `architecture=${MASTER_ARCH:-"amd64"}` line, then the `architecture` will be filled to install-config.yaml for control plane and worker. https://github.com/openshift/release/blob/56d4f807f6a1ef4505d77892cde4b5a0ba444f37/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh#L102, I think the var actually is only severed for control plane.

Reply:
Didn't see that. Changed to $CLUSTER_ARCH, wdyt?

Reply:
sounds okay for me.

Reply:
Sounds good to me, pushing

Reply:
@jianlinliu PTAL

Reply:
lgtm